app/Console/Commands/SendTestEmail.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Mail;
+
+class SendTestEmail extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'monica:test-email
+                            {--email= : The email address to send to}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send a test email to check that delivery is working';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // retrieve the email from the option
+        $email = $this->option('email');
+
+        // if no email was passed to the option, prompt the user to enter the email
+        if (! $email) {
+            $email = $this->ask('What email address should I send the test email to?');
+        }
+
+        // only perform basic validation, the mail server can handle the rest
+        if (strpos($email, '@') === false) {
+            $this->error('Invalid email address - missing "@" symbol!');
+
+            return -1;
+        }
+
+        $this->info("Preparing and sending email to '".$email."'");
```

Comment:
No need for concatenate here:
```suggestion
        $this->info("Preparing and sending email to '$email'");
```